### ISSUE TITLE
Transaction Logging error on instantiating AceBaseLocalSettings

### ISSUE THREAD

**Issue by paradis-A** [appy-one/acebase#74]
I was trying  to use transaction logging then got this error:
TypeError: Cannot set properties of undefined (setting 'transactions')
    at new AceBaseLocalSettings (C:\System\lenlen\lendb-server\node_modules\acebase\src\acebase-local.js:20:42)
    at new AceBase (C:\System\lenlen\lendb-server\node_modules\acebase\src\acebase-local.js:33:19)
    at Object.<anonymous> (C:\System\lenlen\lendb-server\test\refupdates.spec.js:2:12)
    at Module._compile (node:internal/modules/cjs/loader:1097:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1151:10)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47

fortunately i tried to fix it at acebase-local.js  line 19 with this:
```js
if (this.storage && typeof options.transactions === 'object') {
    this.storage.transactions = options.transactions;
}
```
dont know it'll produce bad effect in the code i just need it to propagate sorted query without re-querying all over again.

**Comment by appy-one** [comment]
Thanks, I'll fix that

**Comment by appy-one** [comment]
Fixed in v1.15.1, thanks for reporting!
